Lib9c/Action/Garages/UnloadFromGarages.cs

```diff
@@ -0,0 +1,143 @@
+        {
+            UnloadData = new List<(
+                Address recipientAvatarAddress,
+                IOrderedEnumerable<(Address balanceAddress, FungibleAssetValue value)>?
+                fungibleAssetValues,
+                IOrderedEnumerable<(HashDigest<SHA256> fungibleId, int count)>?
+                fungibleIdAndCounts, string? memo)>();
+        }
+
+        public UnloadFromGarages(
+            IReadOnlyList<(
+                Address recipientAvatarAddress,
+                IEnumerable<(Address balanceAddress, FungibleAssetValue value)>?
+                fungibleAssetValues,
+                IEnumerable<(HashDigest<SHA256> fungibleId, int count)>? fungibleIdAndCounts,
+                string? memo)> unloadData)
+        {
+            UnloadData = unloadData.Select(data => (
+                data.recipientAvatarAddress,
+                GarageUtils.MergeAndSort(data.fungibleAssetValues),
+                GarageUtils.MergeAndSort(data.fungibleIdAndCounts),
+                data.memo)).ToImmutableList();
+        }
+
+        protected override IImmutableDictionary<string, IValue> PlainValueInternal =>
+            new Dictionary<string, IValue>
+            {
+                {
+                    "l",
+                    new List(UnloadData.Select(data =>
+                        new List(data.recipientAvatarAddress.Serialize(),
+                            SerializeFungibleAssetValues(data.fungibleAssetValues),
+                            SerializeFungibleIdAndCounts(data.fungibleIdAndCounts),
+                            string.IsNullOrEmpty(data.memo) ? Null.Value : (Text)data.memo)))
+                }
+            }.ToImmutableDictionary();
+
+        protected override void LoadPlainValueInternal(
+            IImmutableDictionary<string, IValue> plainValue)
+        {
+            var serialized = plainValue["l"];
+            if (serialized is null or Null)
+            {
+                throw new ArgumentNullException(nameof(serialized));
+            }
+
+            if (serialized is not List list)
+            {
+                throw new ArgumentException(
+                    $"The type of {nameof(serialized)} must be bencodex list.");
+            }
+
+            UnloadData = list.Select(rawValue =>
+            {
+                var value = rawValue as List
+                            ?? throw new ArgumentException(
+                                $"The type of {nameof(rawValue)} must be bencodex list.");
+                var recipientAvatarAddress = value[0].ToAddress();
+                var fungibleAssetValues =
+                    GarageUtils.MergeAndSort(
+                        (value[1] as List)?.Select(raw =>
+                        {
+                            var fungibleAssetValue = (List)raw;
+                            return (fungibleAssetValue[0].ToAddress(),
+                                fungibleAssetValue[1].ToFungibleAssetValue());
+                        }));
```

Comment:
It will be treated as `null` instead of throwing an error, even if invalid items like `Dictionary` became.